Sources/TensorFlow/Operators/Math.swift

```diff
@@ -33,26 +33,6 @@ func pow<T: BinaryFloatingPoint>(_ x: T, _ y: T) -> T {
 
 extension Tensor: VectorProtocol where Scalar: Numeric {
     public typealias VectorSpaceScalar = Scalar
-
-    /// Multiplies the scalar with every scalar of the tensor and produces the product.
-    @inlinable
-    @differentiable(vjp: _vjpMultiply(lhs:rhs:) where Scalar: TensorFlowFloatingPoint)
-    public static func * (lhs: Scalar, rhs: Tensor) -> Tensor {
-        return Tensor(lhs) * rhs
-    }
-}
-
-internal extension Tensor where Scalar: TensorFlowFloatingPoint {
-    @inlinable
-    static func _vjpMultiply(lhs: Tensor, rhs: Tensor) -> (Tensor, (Tensor) -> (Tensor, Tensor)) {
```

Comment:
This shouldn't have been placed here, since it has nothing to do with the scalar-tensor multiplication operator.